### PR TITLE
Update helm2 to helm3

### LINKED ISSUE #920: Move helm version to 3.x in install-helm.sh
Helm v3 no longer requires any in cluster components (like tiller) and makes it quite easy to use as a deployment mechanism (no kubernetes components need to be deployed). I suggest that we shift to the same to make the script easier to maintain.

For helm v3 the install command for any chart needs to change like so,

```
-    "${HELM}" install "${SCRIPT_DIR}"/../charts/ceph-csi-cephfs --name ${CEPHFS_CHART_NAME} --namespace ${NAMESPACE} --set provisioner.fullnameOverride=csi-cephfsplugin-provisioner --set nodeplugin.fullnameOverride=csi-cephfsplugin --set configMapName=ceph-csi-config --set provisioner.podSecurityPolicy.enabled=true --set nodeplugin.podSecurityPolicy.enabled=true
+    "${HELM}" install --namespace ${NAMESPACE} --set provisioner.fullnameOverride=csi-cephfsplugin-provisioner --set nodeplugin.fullnameOverride=csi-cephfsplugin --set configMapName=ceph-csi-config --set provisioner.podSecurityPolicy.enabled=true --set nodeplugin.podSecurityPolicy.enabled=true ${CEPHFS_CHART_NAME} "${SCRIPT_DIR}"/../charts/ceph-csi-cephfs
```

IOW, no `--name` parameter and also the chart path and name move to the end of the CLI.

I would suggest that we change the script to require v3 as the minimum and not write code to support both, as the intent is to keep the code simpler with this change.

### PR DESCRIPTION
**Pull request by Madhu-1**
Updated scripts and the travis.yaml to install helm3 to install cephcsi charts and to do helm lint

fixes #920
    
Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
@humblec @nixpanic @ShyamsundarR can you please review this?

File: charts/ceph-csi-cephfs/README.md [hunk 1]
```diff
@@ -6,22 +6,46 @@ The ceph-csi-cephfs chart adds cephfs volume support to your cluster.
...
 
+- For helm 2.x
+
 ```bash
 helm install --namespace "ceph-csi-cephfs" --name "ceph-csi-cephfs" ceph-csi/ceph-csi-cephfs
 ```
 
+- For helm 3.x
+
+If you want to create the namespace, use this command
```
Comment (nixpanic) [inline charts/ceph-csi-cephfs/README.md:17]: This seems to be a hard requirement, right? Maybe phrase it like

> Create the namespace where Helm should install the components with:

File: charts/ceph-csi-cephfs/README.md [hunk 2]
```diff
@@ -6,22 +6,46 @@ The ceph-csi-cephfs chart adds cephfs volume support to your cluster.
...
+
+```bash
+kubectl create namespace ceph-csi-cephfs
+```
+
```
Comment (nixpanic) [inline charts/ceph-csi-cephfs/README.md:22]: Add
> Run the installation:

File: charts/ceph-csi-cephfs/README.md [hunk 3]
```diff
@@ -6,22 +6,46 @@ The ceph-csi-cephfs chart adds cephfs volume support to your cluster.
...
 
 ```bash
 helm status "ceph-csi-cephfs"
 ```
 
+## Delete Chart
+
 If you want to delete your Chart, use this command
 
+- For helm 3.x
```
Comment (nixpanic) [inline charts/ceph-csi-cephfs/README.md:37]: Isn't this the Helm 2.x command?

File: charts/ceph-csi-cephfs/README.md [hunk 4]
```diff
@@ -6,22 +6,46 @@ The ceph-csi-cephfs chart adds cephfs volume support to your cluster.
 
 To install the Chart into your Kubernetes cluster
 
+- For helm 2.x
+
 ```bash
```
Comment (nixpanic) [inline charts/ceph-csi-cephfs/README.md:11]: By adding the `-` you just created a list that requires indention. Make sure to indent the the `\```bash` and lines below.

I suggest to use hackmd.io for checking the layout.

**Comment by Madhu-1** [comment]
@nixpanic changes done PTAL

**Comment by Madhu-1** [comment]
@humblec please review

**Comment by Madhu-1** [comment]
@humblec PTAL

**Pull request merged**